events_worker.go

```diff
@@ -26,21 +26,21 @@ type EventHandler interface {
 	Handle(context.Context, *Event, proto.Message) ([]*Event, ferr.FoundationError)
 }
 
-// ErrorModeEnum defines the EventsWorker behavior when errors occur while handle event
-type ErrorModeEnum int
+// ErrorHandlingStrategy defines the EventsWorker behavior when errors occur while handle event
+type ErrorHandlingStrategy int
 
 const (
-	ErrorModeEnumSkip ErrorModeEnum = iota // Default: commit message and skip the event
-	ErrorModeEnumStop                      // Stop the worker
-	// ErrorModeEnumRetry                  // Retry the event // TODO: implement retry mode
+	IgnoreError     ErrorHandlingStrategy = iota // Default: commit message and skip the event
+	ShutdownOnError                              // Stop the worker
```

Comment:
It better to place comments before the line, not on the same line as the code itself. Easier to read, easier to maintain, no non-relevant `space` garbage in diffs

Reply:
fixed